logging/interceptor.go

```diff
@@ -80,23 +80,49 @@ func UnaryClientInterceptor(logger *logrus.Logger, opts ...Option) grpc.UnaryCli
 		fields[DefaultGRPCCodeKey] = code.String()
 
 		levelLogf(
-			logrus.NewEntry(logger).WithFields(fields),
+			entry.WithFields(fields),
 			options.codeToLevel(code),
-			"finished unary call with code "+code.String())
+			fmt.Sprintf("finished unary call with code %s", code.String()))
 
 		return err
 	}
 }
 
-func UnaryServerInterceptor(logger *logrus.Logger, opts ...Option) grpc.UnaryServerInterceptor {
+func StreamClientInterceptor(entry *logrus.Entry, opts ...Option) grpc.StreamClientInterceptor {
+	options := initOptions(opts)
+
+	return func(ctx context.Context, desc *grpc.StreamDesc, cc *grpc.ClientConn, method string, streamer grpc.Streamer, option ...grpc.CallOption) (grpc.ClientStream, error) {
+		startTime := time.Now()
+		fields := newLoggerFields(method, startTime, DefaultClientKindValue)
+
+		ctx = setInterceptorFields(ctx, fields, entry.Logger, options, startTime)
+
+		clientStream, err := streamer(ctx, desc, cc, method, option...)
+		if err != nil {
+			fields[logrus.ErrorKey] = err
+		}
+
+		code := status.Code(err)
+		fields[DefaultGRPCCodeKey] = code.String()
+
+		levelLogf(
+			entry.WithFields(fields),
+			options.codeToLevel(code),
+			fmt.Sprintf("finished client streaming call with code %s", code.String()))
```

Comment:
Per definition of levelLogf ```func levelLogf(entry *logrus.Entry, level logrus.Level, format string, args ...interface{}) {``` we can add arguments directly into this function instead of using fmt.Sprintf